fundingmanager.go

```diff
@@ -66,6 +57,20 @@ const (
 	minChanFundingSize = btcutil.Amount(20000)
 )
 
+var (
+	// maxFundingAmount is a soft-limit of the maximum channel size
+	// accepted within the Lightning Protocol Currently. This limit is
+	// currently defined in BOLT-0002, and serves as an initial
+	// precautionary limit while implementations are battle tested in the
+	// real world.
+	//
+	// NOTE: This will be increased 60x when running under Litecoin as the
+	// primary chain due to the price difference.
+	//
+	// TODO(roasbeef): add command line param to modify
+	maxFundingAmount = btcutil.Amount(1 << 24)
```

Comment:
IMO it'd be preferable to have enumerate both constants, `maxBtcFundingAmount` and `maxLtcFundingAmount` similar to how we have for the other constraints.

If we want to make this easy to modify in the future, we can also define something like `btcToLtcConversionRate = 60` and compute ltc constants at compile time

Reply:
Fixed.